### DISCUSSION TITLE
Date and time

### DISCUSSION THREAD

**Discussion by sciwonder** [quarto-dev/quarto-cli#1226] [Feature Requests]
Love the look and feel of Quatro! I was just wondering, for blogs, is it currently possible to customise the date element so that it includes date *and time* of publication? Is there a "last updated" element as well? I can imagine some circumstances where displaying the original publication date/time and the "last updated" date/time close together could be useful.

**Comment by cderv** [comment]
Did you already have a look in the documentation about Date and Time within Quarto : https://quarto.org/docs/reference/dates.html ? 

Especially, there is a `last-modified` alias available. 

If this is not working as documented, please share some more details ! Thank you !

**Comment by sciwonder** [comment]
Sorry, I should have made my question a bit clearer. On my previous blog, there were two date fields: `date` and `lastmod`. The first would be the date published and the second would be the date the post was last updated (if applicable), and both were displayed. For example, the following YAML was translated into the output below.

```
---
title: An example post
authors: 
  - Author Name
date: 2019-08-23
lastmod: 2019-09-02
---
```
<img width="500" alt="Screen Shot 2022-06-29 at 6 46 44 pm" src="https://user-images.githubusercontent.com/106318981/176393890-e45bb547-f384-4cc2-b1ea-683294081c66.png">

Currently, in Quatro, the same YAML translates into this output, but `lastmod` is not represented anywhere in the output.

<img width="768" alt="Screen Shot 2022-06-29 at 6 50 48 pm" src="https://user-images.githubusercontent.com/106318981/176394771-af277954-2591-4a62-bd65-1f760cb8f662.png">

Maybe I have missed something, but I think it would be useful to have the option of displaying the date a post was last updated in addition to the date it was published, in cases where those two dates might be different and it might be important for readers to know that. I don't think this should be a compulsory or "default" feature, but it would be nice to have the option as there are circumstances where it would be useful.

**Reply by cderv** [reply]
Oh yes, I don't think we have that. @dragonstyle do we ?

**Reply by dragonstyle** [reply]
We do not currently have this, but it is a nice idea. I will convert this to an issue and track this as an item to complete post 1.0 release.

**Reply by cderv** [reply]
Awesome. Work is now tracked in https://github.com/quarto-dev/quarto-cli/issues/1241

**Comment by mcanouil** [comment]
Closing as completed and in favour of https://github.com/quarto-dev/quarto-cli/issues/1241.
